README.md

```diff
@@ -76,10 +77,17 @@ mvn install
 ````
 
 ### Compile & Build the Proxy
+
 ```bash
 git clone https://github.com/clarus-proxy/proxy.git
 cd proxy
 git checkout develop
-cd proxy
 mvn install
 ````
+
+### Get a test policy and launch the proxy !
```

Comment:
I think that for the sake of Windows users we need to describe the proxy.bat usage.

Reply:
Actually, I though we could get rid of  `proxy.bat`. I didn't remove it in my PR because I tought I may have another unkown usage but when it comes to launching the proxy, it looks useless. This script only forwards CLI arguments to the main proxy jar which seems able to do the argument parsing on its own.

Nonetheless I agree that the README should also mention Windows commands. I will incorporate that in a commit.